### PR TITLE
feat: Clear Regions updates [AB#17269]

### PR DESCRIPTION
**Pull request by tbarlow12**
- Clear regions now has a confirmation modal before actually performing the action
- Ctrl+Delete now triggers the confirmation modal
- Icon for clearing regions is now the Ghostbuster's "ban" icon

### CODE REVIEW

File: src/registerToolbar.ts [hunk 1]
```diff
@@ -86,9 +86,10 @@ export default function registerToolbar() {
     ToolbarItemFactory.register({
         name: ToolbarItemName.ClearRegions,
         tooltip: "Clear Regions",
-        icon: "fa-trash-alt",
+        icon: "fa-ban",
         group: ToolbarItemGroup.Regions,
         type: ToolbarItemType.Action,
+        accelerators: ["Ctrl+Delete"],
```
Comment (mydiemho) [inline src/registerToolbar.ts]: I thought this only remove selected regions, or does it remove all?

Comment (tbarlow12) [reply src/registerToolbar.ts]: Delete would remove selected regions, ctrl delete removes all

File: src/react/components/pages/editorPage/canvasHelpers.test.ts [hunk 2]
```diff
@@ -105,7 +105,7 @@ describe("Canvas Helpers", () => {
 
     it("Duplicates and moves a region", () => {
         const regions = MockFactory.createTestRegions();
-        const duplicates = CanvasHelpers.duplicateRegionsAndMove([regions[0]], regions, 1000, 2000);
+        const duplicates = CanvasHelpers.duplicateRegionsAndMove([regions[0]], regions, 10000, 10000);
```
Comment (PIC123) [inline src/react/components/pages/editorPage/canvasHelpers.test.ts]: why did you need to increase the magnitude of these?
